core/commands/active.go

```diff
@@ -22,7 +22,7 @@ Lists running and recently run commands.
 		res.SetOutput(req.InvocContext().ReqLog.Report())
 	},
 	Options: []cmds.Option{
-		cmds.BoolOption("v", "verbose", "print more verbose output"),
+		cmds.BoolOption("v", "verbose", "print more verbose output").Default(false),
```

Comment:
Going to assume that yes, we should add `Default(false)` from now on. See https://github.com/ipfs/go-ipfs/pull/2582#issuecomment-214925152